projects/plugins/boost/CHANGELOG.md

```diff
@@ -125,3 +138,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.3.1-beta]: https://github.com/Automattic/jetpack-boost-production/compare/v1.3.0-beta...v1.3.1-beta
 [1.3.0-beta]: https://github.com/Automattic/jetpack-boost-production/compare/v1.2.0...v1.3.0-beta
 [1.2.0]: https://github.com/Automattic/jetpack-boost-production/compare/v1.1.0...v1.2.0-beta
+[2.0.0-beta]: https://github.com/Automattic/jetpack-boost-production/compare/v1.4.0...v2.0.0-beta
```

Comment:
I think this should be `1.4.1-beta`, and should be above the other links?

Reply:
Yes, it should be. Fixed in 8fd2204189. Thank you